analysis_notebooks/Gender Analysis.ipynb

```diff
@@ -63,12 +63,15 @@
    "source": [
     "from __future__ import print_function\n",
     "import argparse\n",
-    "import datasets\n",
+    "import os\n",
+    "try: \n",
+    "    import datasets\n",
+    "except ModuleNotFoundError: # switch to parent directory on first run \n",
```

Comment:
why do you need to try this, shouldn't you just automatically switch into parent directory? also, datasets will never be imported since it will fail here right?

Reply:
Yes I will add the import statement to the except clause as well. I was trying to control for the scenario if someone accidentally re-runs the cell, then it will keep exiting out to parent directory.

Reply:
good call!